skimage/registration/_lddmm.py

```diff
@@ -0,0 +1,873 @@
+import numpy as np
+from scipy.interpolate import interpn
+from scipy.linalg import inv, solve, det
+from skimage.transform import resize
+from matplotlib import pyplot as plt
```

Comment:
not needed I think

Reply:
I've removed the resize import, the rest are used.